shell/components/Tabbed/Tab.vue

```diff
@@ -32,6 +32,11 @@ export default {
       type:    Boolean,
       default: null
     },
+    // ToDo: this is redundant with the above, fix that...
```

Comment:
Can we make sure to have an associated issue mentioned inline with the todo?

Reply:
I agree. I think we shouldn't have any ToDos in the code itself because the GitHub issues are more organized.

Reply:
Looking at this closer, these two props aren't perfectly redundant. `displayAlertIcon` will always show an alert icon whereas `error` will only show an error if the tab is not currently active. Removing the ToDo for now.